### DISCUSSION TITLE
Problems connecting Thunderbird to  mailbox

### DISCUSSION THREAD

**Discussion by draconMo** [modoboa/modoboa#3363] [Help]
Debian
Modoboa 2.2.0
Thunderbird Version 128.5.1esr (64-bit)
Setting up a connection to a mailbox via imap

![thuderbird](https://github.com/user-attachments/assets/e4f802f2-e160-43ba-82c0-d9d485cb01a8)

All data is entered correctly. 
I press Re-test. 
can't connect
postfix log

mail postfix/submission/smtpd[3062896]: connect from test.rowen.ru[193.105.37.198]
mail postfix/submission/smtpd[3062896]: improper command pipelining after EHLO from test.rowen.ru[193.105.37.198]: QUIT\r\n
mail postfix/submission/smtpd[3062896]: disconnect from test.rowen.ru[193.105.37.198] ehlo=0/1 commands=0/1

Moreover, if you continue connecting without Re-test. Then the mailbox will be created and receiving and sending mail will work. 

Tell me, is this a postfix configuration problem? Or is this a thunderbird problem?

**Comment by cotarconsorcio** [comment]
Estou com o mesmo problema. Você conseguiu resolver?
